monitoring/onpremise/seq/main.tf

```diff
@@ -69,6 +69,13 @@ resource "kubernetes_deployment" "seq" {
               value = "FMB0CwtRt8CwkiSDebSmdJszUzK9B52DV19CKdpFyGtrGRkBrQ=="
             }
           }
+          dynamic "env" {
+            for_each = var.authentication ? [false] : [true]
+            content {
+              name  = "SEQ_FIRSTRUN_NOAUTHENTICATION"
+              value = "true"
+            }
+          }
```

Comment:
The block should not be dynamic. But the value should depend on `var.authentication`.